src/sagemaker/jumpstart/curated_hub/utils.py

```diff
@@ -164,3 +179,97 @@ def create_hub_bucket_if_it_does_not_exist(
     )
 
     return bucket_name
+
+def tag_hub_content(hub_content_arn: str, tags: List[Tag], session: Session) -> List[Dict[str, List[Dict[str, str]]]]:
+    responses = []          
+    for tag in tags:
+      responses.add(session.add_tags(
```

Comment:
It's not clear to me why we need to return the responses. Can we log the success after the `for` loop?

Reply:
We could - I added in the response here since it could be possible something is dropped along the way. The Tagris response should be the SOT for what tags were actually added